docs/pages/project/contributing/contributing-docs.md

```diff
@@ -248,20 +248,23 @@ Whenever the code tags are detected, the clipboard javascript file is automatica
 
 ## Documentation Contribution Flow Summary
 
+**NOTE: For contributing `mesheryctl` reference section, refer [Contributing CLI](/contributing-cli)**
+
+
 The following is a concise summary of the steps to contribute to Meshery documentation.
 
 1. Create a fork, if you have not already, by following the steps described [here](CONTRIBUTING-gitflow.md)
-1. In the local copy of your fork, navigate to the docs folder.
+2. In the local copy of your fork, navigate to the docs folder.
```

Comment:
Generally speaking, auto-numbering with `1.` is preferred.

Reply:
👍